packages/web-client/src/helpers/space/functions.ts

```diff
@@ -0,0 +1,128 @@
+import { User } from '../user'
```

Comment:
What about a file name like e.g. `conversion.ts`? 🤔  For now the only function that this file has is a conversion from a drive response to a resource. On the other hand we could build a class for the space resource in a followup... your choice.

Reply:
I find conversion strange tbh, it's more of a deserialize function to me ... then again the function is called buildSpace ... which is also a bit of a weird name, so I rather wouldn't call the file `build.ts` :shrug: